src/oc_span_sweeper.erl

```diff
@@ -88,6 +77,45 @@ terminate(_Reason, _State, _Data) ->
 
 %%
 
+do_gc(#data{strategy=Strategy,
+            ttl=TTL,
+            storage_size=MaxSize}) ->
+
+    StorageSize = case MaxSize of
+                      infinity ->
```

Comment:
Not a big deal, but personally I'd find it cleaner to be like:

```erlang
do_gc(#data{strategy=Strategy,
            ttl=TTL,
            storage_size=infinity}) ->
    sweep_spans(Strategy, TTL);
do_gc(#data{strategy=Strategy,
            ttl=TTL,
            storage_size=MaxSize}) ->
    StorageSize = ets:info(?SPAN_TAB, memory) * erlang:system_info({wordsize, external}),
    ...
```

Reply:
was thinking about this, thought would be ugly, but now I see it's not